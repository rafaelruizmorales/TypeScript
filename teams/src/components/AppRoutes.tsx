import * as React from 'react'
import {Route, Switch} from 'react-router-dom'
import {PlayersPage} from '../pages/view-players/PlayersPage'
import {ProfilePage} from '../pages/view-profile/ProfilePage'
import {TeamsPage} from '../pages/view-teams/TeamsPage'

export const AppRoutes: React.FC = () => {
  return (
    <Switch>
      <Route path="/teams">
        <TeamsPage />
      </Route>
      <Route path="/profile">
        <ProfilePage />
      </Route>
      <Route path="/">
        <PlayersPage />
      </Route>
    </Switch>
  )
}
