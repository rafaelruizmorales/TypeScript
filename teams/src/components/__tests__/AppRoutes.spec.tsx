import {shallow} from 'enzyme'
import * as React from 'react'
import {AppRoutes} from '../AppRoutes'

describe('AppRoutes', () => {
  it('should match the snapshot', () => {
    expect(shallow(<AppRoutes />)).toMatchSnapshot()
  })
})
