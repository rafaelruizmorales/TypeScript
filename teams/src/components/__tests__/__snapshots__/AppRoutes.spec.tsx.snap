// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRoutes should match the snapshot 1`] = `
<Switch>
  <Route
    path="/teams"
  >
    <TeamsPage />
  </Route>
  <Route
    path="/profile"
  >
    <ProfilePage />
  </Route>
  <Route
    path="/"
  >
    <PlayersPage />
  </Route>
</Switch>
`;
